llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1448,3 +1448,18 @@ def FeatureTaggedGlobals : SubtargetFeature<"tagged-globals",
     "AllowTaggedGlobals",
     "true", "Use an instruction sequence for taking the address of a global "
     "that allows a memory tag in the upper address bits">;
+
+def FeatureVendorMIPSCMove : SubtargetFeature<"xmipscmove", "HasVendorMIPSCMove",
```

Comment:
FeatureVendorMIPSCMove -> FeatureVendorXMIPSCMove so it matches the name of the extension